### PR TITLE
add test for oasis statistics

### PR DESCRIPTION
**Pull request by ObiWahn**
add test for oasis

### CODE REVIEW

**Comment by ObiWahn** [comment]
We need to wait until `permissions_server` tests are enabled in oskar pr-runs.

File: tests/js/server/permissions/test-oasis.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/*jshint globalstrict:false, strict:false */
+/* global getOptions, assertTrue, assertFalse, assertEqual, assertMatch, fail, arango */
```
Comment (jsteemann) [inline tests/js/server/permissions/test-oasis.js]: it seems this line is copy&paste. Most of these symbols are not used in this file.

**Comment by jsteemann** [comment]
Please start a Jenkins PR run for htis PR.

**Comment by ObiWahn** [comment]
> Please start a Jenkins PR run for htis PR.

I'll do when vadim tells me the tests are enabled for the cluster. Otherwise nothing would be tested.

**Comment by jsteemann** [comment]
Wouldn't the test run in single server as well with the current Jenkins setup?
We need to make sure that it also works in single server, don't we?

**Comment by jsteemann** [comment]
Plus that there are no jslint errors etc.
Starting a PR run is always good.

File: tests/js/server/permissions/test-oasis.js (hunk 1 above)
Comment (ObiWahn) [reply tests/js/server/permissions/test-oasis.js]: Removed `arango` the others are global functions (`assertFalse`, `assertMatch`, `fail`) will probably be used sooner or later in this file.

**Comment by ObiWahn** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/5095/

**Pull request merged**